documentation/modules/ROOT/pages/configuration/signalling.adoc

```diff
@@ -0,0 +1,124 @@
+// Category: debezium-using
+// Type: assembly
+// ModuleID: customization-of-kafka-connect-automatic-topic-creation
+// Title: Sending signals to a Debezium connector
+[id="sending-signals-to-a-debezium-connector"]
+= Sending signals to a Debezium connector
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+[NOTE]
+====
+This feature is currently in incubating state, i.e. exact semantics, configuration options etc. may change in future revisions, based on the feedback we receive. Please let us know if you encounter any problems while using this extension.
+====
+
+== Overview
+Sometimes it is necessary for the user to modify a connector behaviour or trigger a one-time action (e.g. snapshot of a single table).
+To fullwfill such needs {prodname} provides a mechanism how to send a signal to a {prodname} connector to trigger an action.
+As it might be necessary to synchronize such an action with the dataflow the signal is implemented as a write to a data collection.
+
+Signalling is disabled by default.
+The name of the signalling data collection must be set via connector configuration parameter to enable the function.
+The signalling data collection must be *explictly* added among captured data collections.
```

Comment:
Is this to say it must be part of the include list explictly? If so, I think we should change this to capture that table as soon as the `signal.data.collection` parameter is provided.

Reply:
@gunnarmorling  Yes, I considered this approach too and I am inclined to change the behaviour in future.
Right now I am unsure if we want to play with filters behind users back.
Moreover such change would not require backward compatibility handling.

Reply:
Ok, captured this in https://issues.redhat.com/browse/DBZ-3293. We definitely can simplify the user's life here, so let's do that.